.vscodeignore

```diff
@@ -1,7 +1,9 @@
+.github/**
```

Comment:
Potential improvement:

In other projects, I usually [copy over](https://github.com/salesforce-misc/bazelrc-lsp/blob/aaf7ed6662c8d426449a92687a40c24737fc5181/vscode-extension/build.js#L22) the relevant files into the `dist` folder and then [call vsce on that subfolder](https://github.com/salesforce-misc/bazelrc-lsp/blob/aaf7ed6662c8d426449a92687a40c24737fc5181/vscode-extension/package.json#L46)

Thereby, I have to opt-in each included file, instead of having to opt-out via the `.vscodeignore` file, thereby avoiding unintended bloat in the packaged extension

Let me know what you think :)

Reply:
SG. I've also removed `.vscodeignore`.